proxy/ParentSelection.cc

```diff
@@ -524,6 +557,7 @@ ParentRecord::Init(matcher_line *line_info)
   const char *tmp;
   char *label;
   char *val;
+  char parent_buf[8192]  = {0};
```

Comment:
Yeah I bumped it up.  MAX_PARENTS is 64.  64 * 255 bytes (max fqdn length) = 16320 + a few bytes for the separator.